content/_code-samples/issue-a-token/js/issue-a-token.js

```diff
@@ -0,0 +1,194 @@
+// Stand-alone code sample for the "issue a token" tutorial:
+// https://xrpl.org/issue-a-fungible-token.html
+
+// Uncomment these dependencies for Node.js. In browsers, use <script> tags as
+// in the example demo.html.
+// const ripple = require('ripple-lib')
+// const fetch = require('node-fetch')
+// const submit_and_verify = require('../submit-and-verify/submit-and-verify.js').submit_and_verify
+
+// Connect ---------------------------------------------------------------------
+async function main() {
+  api = new ripple.RippleAPI({server: 'wss://s.altnet.rippletest.net:51233'})
+  console.log("Connecting to Testnet...")
+  await api.connect()
+
+  // Get credentials from the Testnet Faucet -----------------------------------
+  // This doesn't technically need to happen after you call api.connect() but
+  // it's convenient to do here.
+  async function get_faucet_address() {
+    const faucet_url = "https://faucet.altnet.rippletest.net/accounts"
```

Comment:
There's a built-in method to do this now in ripple-lib: https://github.com/ripple/ripple-lib/pull/1497

Reply:
Updated the samples to use the new function.